### PR TITLE
docs: Create read me

### PR DESCRIPTION
**Pull request by CrowRising**
Made a read me from our collective notes in our project board and am RE-DOING this coreectly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
-https://docs.google.com/document/d/1mXX4xO6k7aFdUO3jWYz3Tc7ONjrx3_nEftG3Vrv3wkc/edit?usp=sharing# Futbol
+# Futbol 
+### brought to you by DefCom4
```
Comment (thawley2) [inline README.md:2]: The best name ever!

**Review by angelbyun — APPROVED** [review]
Looks great! Thanks for getting this set up.

File: README.md (hunk 1 above)
Comment (CrowRising) [reply README.md:2]: merci

**Review by andrew-bingham1 — APPROVED** [review]
Looks great to me!

**Review by thawley2 — APPROVED** [review]
Thanks for setting this up Crow!!!

**Pull request merged**